Mathlib/MeasureTheory/Measure/Real.lean

```diff
@@ -0,0 +1,422 @@
+/-
+Copyright (c) 2025 Sébastien Gouëzel. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sébastien Gouëzel
+-/
+import Mathlib.MeasureTheory.Measure.Prod
+import Mathlib.Tactic.Finiteness
+
+/-!
+# Measures as real valued-functions
+
+Given a measure `μ`, we define `μ.real` as the function sending a set `s` to `(μ s).toReal`, and
+develop a basic API around this.
+
+We essentially copy relevant lemmas from the files `MeasureSpaceDef.lean`, `NullMeasurable.lean` and
+`MeasureSpace.lean`, and adapt them by replacing in their name `measure` with `measureReal`.
+
+Many lemmas require an assumption that some set has finite measure. These assumptions are written
+in the form `(h : μ s ≠ ∞ := by finiteness)`, where `finiteness` is a tactic for goals of
+the form `≠ ∞`.
+
+There are certainly many missing lemmas. The missing ones should be added as they are needed.
+
+There are no lemmas on infinite sums, as summability issues are really
+more painful with reals than nonnegative extended reals. They should probably be added in the long
+run.
+-/
+
+open MeasureTheory Measure Set
+open scoped ENNReal NNReal Function symmDiff
+
+namespace MeasureTheory
+
+variable {α β ι : Type*} {_ : MeasurableSpace α} {μ : Measure α} {s s₁ s₂ s₃ t t₁ t₂ u : Set α}
+
+variable (μ) in
+/-- The real-valued version of a measure. Maps infinite measure sets to zero. Use as `μ.real s`. -/
+protected def Measure.real (s : Set α) : ℝ :=
+  (μ s).toReal
+
+theorem measureReal_def (s : Set α) : μ.real s = (μ s).toReal := rfl
```

Comment:
Should this be
```suggestion
theorem Measure.real_def (s : Set α) : μ.real s = (μ s).toReal := rfl
```
since then you can use `rw [μ.real_def]`?

Reply:
All the other lemmas are spellt using `measureReal`, so it looks more natural to me to use the same spelling here for coherence. But I agree that the other version is nice because it enables dot notation. Let me add an alias.